static/app/views/explore/logs/logsTable.tsx

```diff
@@ -79,7 +79,7 @@ export function LogsTable({
 
   return (
     <Fragment>
-      <Table ref={tableRef} styles={initialTableStyles}>
+      <Table ref={tableRef} styles={initialTableStyles} data-test-id="logs-table">
```

Comment:
We don't need test-id anymore? It's fine to leave it, I'll clean it up when I change tables for autorefresh